mp/src/game/client/neo/ui/neo_hud_ghost_marker.cpp

```diff
@@ -129,13 +124,38 @@ void CNEOHud_GhostMarker::DrawNeoHudElement()
 		}
 	}
 
-	surface()->DrawSetColor(ghostColor);
-	surface()->DrawSetTexture(m_hTex);
-	surface()->DrawTexturedRect(
-		offset_X,
-		offset_Y,
-		offset_X + (m_iMarkerTexWidth * scale),
-		offset_Y + (m_iMarkerTexHeight * scale));
+	const float scale = neo_ghost_marker_hud_scale_factor.GetFloat();
```

Comment:
Also, change the default of the scale factor. It looked very small with 0.125 default.